doc/nrf/device_guides/working_with_nrf/nrf91/nrf9160_gs.rst

```diff
@@ -133,6 +134,14 @@ Connecting the |DK| to nRF Cloud
 To transmit data from your |DK| to nRF Cloud, you need an `nRF Cloud`_ account.
 nRF Cloud is Nordic Semiconductor's platform for connecting your IoT devices to the cloud, viewing and analyzing device message data, prototyping ideas that use Nordic Semiconductor's chips, and more.
 
+Once you have an account, activate your SIM card and add the |DK| to your nRF Cloud account.
+
+.. note::
+
+   The instructions assume you are activating an iBasis SIM card that came with your |DK|.
+
+   If you are using a SIM card from another provider, make sure you activate it through your network operator before adding the device to nRF Cloud.
```

Comment:
Maybe indicate which step/heading to skip to if not activating an iBasis SIM?

Reply:
I initially tried to add a link to the relevant section, but the link breaks inside the note when retrieved by the include on the Thingy:91 page. And I don't want to directly name the section without making it into a link.